### DISCUSSION TITLE
Suggest:  Undo, redo

### DISCUSSION THREAD

**Discussion by darco1991** [TiagoSilvaPereira/vemto-issues#388] [Ideas & Suggestions]
Hello @TiagoSilvaPereira , I love 'Vemto', it's bacansisimo, but you could add back buttons for actions, honestly, some of us are a disaster and we make chaos when we edit :(.

**Comment by TiagoSilvaPereira** [comment]
Hi @darco1991 thank you for your suggestion, I'll move it to the discussions channel
